### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by tanweijie123**
- Remove dead codes (Stage and Scene) from ClientInfoPage
- Refactor UniqueTList -> UniqueList & isExisting -> isUnique #76 
- Change application icon (calendar -> muscle) 
- Add autocomplete to command box #100 
- [Draft] Homepage middle section to display today's schedule #31 
- Change HelpWindow design and utility
- Modify Command TextField

Changed Icon 
image

Auto-complete snip: 
image
*Side effect: If the user presses "tab", it will auto-complete the first line 

Homepage today's schedule (Draft) 
image
_Future fixes could include a nicer UI to display schedule. To be discussed_

HelpWindow Changes
image
_By clicking on the link itself, it will open the UG in a browser. **Requires testing on Linux and MacOS**_
- Pressing Enter will focus on the link and 'click' it. 
- Pressing ESC will close the window. 

Modified Command TextField
image

### CODE REVIEW

**Comment by benclmnt** [comment]
Have we checked with Prof Damith on using controlsfx as external libraries?

File: src/main/resources/view/DarkTheme.css [hunk 1]
```diff
@@ -97,14 +101,15 @@
     -fx-label-padding: 0 0 0 0;
     -fx-graphic-text-gap : 0;
     -fx-padding: 0 0 0 0;
+    -fx-text-fill: white;
 }
 
 .list-cell:filled:even {
-    -fx-background-color: #3c3e3f;
+    -fx-background-color: #3c3e3f !important;
```
Comment (benclmnt) [inline src/main/resources/view/DarkTheme.css:108]: hmm, what happens if we remove `!important` here? 🤔

**Review by benclmnt — APPROVED** [review]
Looks good to me!

**Comment by tanweijie123** [comment]
> Have we checked with Prof Damith on using controlsfx as external libraries?

Yup. Approved. 
https://github.com/nus-cs2103-AY2021S1/forum/issues/308

File: src/main/resources/view/DarkTheme.css (hunk 1 above)
Comment (tanweijie123) [reply src/main/resources/view/DarkTheme.css:108]: It is a quick fix for AutoComplete. So without it, we must find the cause of why the first row was different colouring

image

**Pull request merged**